### PR TITLE
Rounds all organ health damage to the second decimal

### PR DESCRIPTION
**Pull request by HMBGERDO**
## What Does This PR Do

This PR makes all organ damage numbers fixed to 2 symbols after the period

## Why It's Good For The Game

Receiving 0.2359 damage because funny armor blocked some part of it is not really better than receiving just 0.24 damage
0.2359 damage makes all health texts unpredictable while that accuracy is not neccesary
This just makes people's life better, dunno what to say else

## Testing

Compiled, received some damage, healed

## Changelog
:cl:
tweak: All external and internal organs damage is now limited to 2 symbols after the period
/:cl:

### CODE REVIEW

**Comment by Danchi299** [comment]
I think PR title is a bit indescriptive

**Comment by HMBGERDO** [comment]
Corrected PR name, i hope this is better

File: code/modules/surgery/organs/organ.dm [hunk 1]
```diff
@@ -217,6 +217,7 @@
 	if(tough)
 		return
 	damage = clamp(damage + amount, 0, max_damage)
+	damage = round(damage, 0.1)
```
Comment (farie82) [inline code/modules/surgery/organs/organ.dm:220]: Would be good to make a proc for this somewhere that can be used by the rest. Maybe also make it round to `0.01` as I'm not 100% sure if `0.1` wouldn't cause weirdness with certain chems/virus symptoms/etc

Comment (HMBGERDO) [reply code/modules/surgery/organs/organ.dm]: done

**Comment by TravisAngeI** [comment]
i think a better name would be

"Rounds all organ health damage to the second decimal"

**Comment by HMBGERDO** [comment]
*deathgasp

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Pull request merged**